addons/mail/static/src/new/thread/attachment_list.js

```diff
@@ -14,10 +14,11 @@ import { _t } from "@web/core/l10n/translation";
  * @property {import("@mail/new/core/attachment_model").Attachment[]} attachments
  * @property {function} unlinkAttachment
  * @property {number} imagesHeight
+ * @property {boolean} canBeEdited
  * @extends {Component<Props, Env>}
  */
 export class AttachmentList extends Component {
-    static props = ["attachments", "unlinkAttachment", "imagesHeight"];
+    static props = ["attachments", "unlinkAttachment", "imagesHeight", "canBeEdited?"];
```

Comment:
`canBeEdited` => `editable`

I took the opportunity of this PR to rename all `canBeEditable` to `editable`, and all `canBeDeleted` to `deletable`